### DISCUSSION TITLE
Manual/SL-Properties

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#311] [Notes]
# Manual/SL-Properties

https://docs.unity3d.com/Manual/SL-Properties.html

**Comment by SoylentGraham** [comment]
There are a lot of options to effect the Inspector for your material documented on [this page](https://docs.unity3d.com/ScriptReference/MaterialPropertyDrawer.html) but not above.

**Comment by alexmalyutindev** [comment]
For **Texture2D** material properties, you can also use **"linearGrey"** as a default value. This provides a 2x2 greyscale texture in **linear color space**, which can be useful for neutral texture inputs like detail maps.

```shader
_MainTex ("Main Texture", 2D) = "linearGrey" {}
```
